bin/local/create-local-user.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+### failsafe logic to exit in case we are not running in our local minikube context
+KUBE_CONTEXT=$(kubectl config current-context)
+
+echo "Current kube context: '${KUBE_CONTEXT}'"
+
+if [[ "${KUBE_CONTEXT}" != "minikube-wbaas" ]]; then
+    echo "Error: wrong kube context. Use this script only within 'minikube-wbaas'!"
+    exit 1
+fi
+#####################################################################################
+
+# This script should create and verify a user account on a local wbaas cluster with the credentials below. Do not use this in production! 
+
+USER_CODE="create-local-user"
+USER_MAIL="jane.doe@wikimedia.de"
+USER_PASS="wikiwikiwiki"
+
+TEST_RESPONSE=$(curl -s http://api.wbaas.localhost/healthz)
```

Comment:
Could we somehow make this work without someone having to `minikube-tunnel` beforehand? I.e. ask Kubernetes to portforward to api and then do the pinging?

Reply:
Yeah it could get a little tricky. Let's resort to the current state for now if you're ok with it too.